cmd/proxy/models/models.go

```diff
@@ -1,22 +1,15 @@
 package models
 
-import (
-	"log"
-
-	"github.com/gobuffalo/pop"
-	"github.com/gomods/athens/pkg/config/env"
-)
-
 // DB is a connection to your database to be used
 // throughout your application.
-var DB *pop.Connection
+// var DB *pop.Connection
```

Comment:
@manugupt1 my memory is pretty rough on this, but I think this has to stay in order for `buffalo build` to work. Would you be able to try out a `buffalo build` and see?

Same thing with the file inside olympus

Reply:
Does `buffalo dev` not build as well? We are running `buffalo dev` in our e2e tests for proxy, I don't think we are testing Olypmus. Maybe we can add another test stage after e2e tests? If so I will open a new issue.

Reply:
`buffalo build` did not work as the binaries are tagged somehow
`buffalo dev` worked